pkg/liqo-controller-manager/namespaceMap-controller/testUtils/namespacemap_controller_test_utils.go

```diff
@@ -0,0 +1,45 @@
+package namespacemapctrltestutils
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	roleBindingName1 = "role-binding-1"
+	roleBindingName2 = "role-binding-2"
+	roleType         = "Role"
+	roleName         = "fake"
+)
+
+// The remote namespace must have at least 2 roleBinding inside.
+
+// GetRoleBinding1ForASpecificNamespace provides a first roleBinding in the namespace passed as parameter.
+func GetRoleBinding1ForASpecificNamespace(namespaceName string) rbacv1.RoleBinding {
```

Comment:
You could join the two functions taking as parameter an index which is added as suffix to the name.

Reply:
thank you